lib/parity/environment.rb

```diff
@@ -72,10 +73,13 @@ def forced?
       arguments.include?("--force")
     end
 
+    def parallelize?
+      arguments.include?("--parallelize") || arguments.include?("-p")
```

Comment:
I think it's fine to only keep the verbose case here. For users where this makes a difference, typing out one full work is going to be a difference of one second for what's going to be an hour of waiting.

Reply:
Sounds good, PR update forthcoming.

Also, I'm having trouble getting the tests to pass in Travis, although they're passing on my local.  I know,

Reply:
The Travis failures look like different expectations for processor count.

Reply:
Cool, I figured it out.  Should be fixed now.